crates/re_selection_panel/src/visualizer_ui.rs

```diff
@@ -28,25 +27,49 @@ pub fn visualizer_ui(
         ui.label(ui.ctx().error_text("Entity not found in view."));
         return;
     };
+    let active_visualizers: Vec<_> = data_result.visualizers.iter().sorted().copied().collect();
+    let available_inactive_visualizers = available_inactive_visualizers(
+        ctx,
+        ctx.recording(),
+        space_view,
+        &data_result,
+        &active_visualizers,
+    );
 
+    ui.large_collapsing_header_with_button(
+        "Visualizers",
+        true,
+        |ui| {
+            visualizer_ui_impl(ctx, ui, &data_result, &active_visualizers);
+        },
+        re_ui::HeaderMenuButton::new(&re_ui::icons::ADD, |ui| {
+            menu_add_new_visualizer(
+                ctx,
+                ui,
+                &data_result,
+                &active_visualizers,
+                &available_inactive_visualizers,
+            );
+        })
+        .with_enabled(!available_inactive_visualizers.is_empty())
+        .with_hover_text("Add additional visualizers.")
+        .with_disabled_hover_text("No additional visualizers available."),
```

Comment:
Our DESIGN.md guideline prescribe no period for short sentence.
```suggestion
        .with_hover_text("Add additional visualizers")
        .with_disabled_hover_text("No additional visualizers available"),
```